Gemfile

```diff
@@ -233,7 +233,7 @@ group :test do
   gem 'json_spec', '~> 1.1.4'
   gem 'shoulda-matchers', '~> 4.5', require: nil
 
-  gem 'parallel_tests', '~> 3.1'
+  gem 'parallel_tests'
```

Comment:
Why did you remove the version pinning here? Usually we try to pin to a minor version to be able to safely upgrade gems

Reply:
Fixed it :)